.babelrc

```diff
@@ -1,10 +1,11 @@
 {
   "presets": [
-    "./build/preset-es2015",
```

Comment:
Use `env` property for preset injection instead of generating expected content in separate file.